### DISCUSSION TITLE
지금 병원 추출 하는게 막혔나요??

### DISCUSSION THREAD

**Discussion by ranware** [changdoc/naver-vaccine-macro#145] [Q&A]
지금 병원 추출 하는게 막혔나요??

코드 오류가 뜨고 있습니다.

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/discussions/147
와 같은 내용인듯한데 내일 오전 되어야 확인 가능할 것 같습니다.

**Reply by changdoc** [reply]
현재 정상이네요. 야간에만 데이터가 비어 있는것 같습니다.

**Reply by ranware** [reply]
아하 감사합니다
